packages/ui/src/molecules/QuantitySelector/QuantitySelector.tsx

```diff
@@ -0,0 +1,96 @@
+import type {
+  AriaAttributes,
+  ReactNode,
+  InputHTMLAttributes,
+  MouseEventHandler,
+} from 'react'
+import React from 'react'
+
+import Input from '../../atoms/Input'
+import IconButton from '../IconButton'
+import { PlusIcon, MinusIcon } from './stories/assets/Icons'
```

Comment:
Not sure if we should import assets from the stories folder since we could need these icons in other components as well. What do you think?